molecule/default/tests/test_default.py

```diff
@@ -1,25 +1,130 @@
 import os
 import re
-
 import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
-@pytest.mark.skip(reason='TODO: will update the test to run '
-                         'cinder command on controller')
+@pytest.mark.jira('asc-222')
+@pytest.mark.skip(reason='this test can not run on storage host')
 def test_cinder_service(host):
+    """
+    TODO: will update the test to run cinder command on controller node
```

Comment:
TODO has a special meaning in PyCharm if that is what you're aiming for: https://www.jetbrains.com/help/pycharm/creating-todo-items.html

It needs to be a comment if you're trying to use a PyCharm specific TODO.